Algorithm.CSharp/BaseBitcoin.cs

```diff
@@ -0,0 +1,87 @@
+﻿using NodaTime;
+using QuantConnect.Brokerages;
+using QuantConnect.Data.Market;
+using QuantConnect.Securities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+
+    /// <summary>
+    /// Base class for Bitcoin algorithms. This base class provides standard behaviour for trading BTCUSD on Bitfinex
+    /// trading on Bitfinex
+    /// </summary>
+    public abstract class BaseBitcoin : QCAlgorithm
```

Comment:
Any new algorithms should not be included in the brokerage implementation pull request, maybe we can split this algos into a separate PR. Also, ideally we wouldn't need a base bitcoin algorithm class.